src/hooks/mxapp.js

```diff
@@ -10,7 +10,8 @@ const parseMXAppTrigger = trigger =>
     }) => {
         const events = [];
 
-        if (message == "UP")
+        if ((message == "UP" && source != "MXControl")
```

Comment:
this condition can be simplified to `source == "MXControl" || message == "UP"`

Reply:
should fail if `source == "MXControl" && message == "UP"`
 -- mxcontrol is an application and not an instance.

`source == "MXControl" || message == "UP"` allows this.

Reply:
fuck it this is dumb